be/src/storage/vectorized/column_predicate_rewriter.cpp

```diff
@@ -336,8 +336,8 @@ bool ColumnPredicateRewriter::_rewrite_expr_predicate(ObjectPool* pool, const Co
     column_ref._type = type_desc;
     Expr* probe_expr = &column_ref;
     // probe_expr will be copied into filter, so we don't need to allocate it.
-    ExprContext* filter =
-            RuntimeFilterHelper::create_runtime_in_filter(state, pool, probe_expr, eq_null, null_in_set, is_not_in);
+    ExprContext* filter = RuntimeFilterHelper::create_runtime_in_filter(state, pool, probe_expr, eq_null, null_in_set,
```

Comment:
shouldn't create runtime filter in column_predicate_rewriter.cpp

Reply:
Yes, I'll refactor this piece of code and create a generic function to create `in-predicate`